app/controllers/application_controller.rb

```diff
@@ -1,2 +1,4 @@
 class ApplicationController < ActionController::Base
+  http_basic_authenticate_with name: ENV["HTTP_BASIC_AUTH_NAME"],
+                               password: ENV["HTTP_BASIC_AUTH_PASSWORD"]
```

Comment:
Let's use `ENV.fetch` so that we ensure these keys are always set 🙏 
```suggestion
  http_basic_authenticate_with name: ENV.fetch"HTTP_BASIC_AUTH_NAME"),
                               password: ENV.fetch("HTTP_BASIC_AUTH_PASSWORD")
```

Reply:
:pray:

Reply:
I would apply your commit suggestion directly, but you're missing a "("

Reply:
uhh sry